main.go

```diff
@@ -88,6 +89,7 @@ var cli struct {
 		CacheSize int    `default:"64" help:"Size of cache in Megabytes."`
 		Bucket    string `help:"Remote bucket"`
 		PublicURL string `help:"Public base URL of tile endpoint for TileJSON e.g. https://example.com/tiles/"`
+		TileEtag  bool   `help:"Generate etag for each tile instead of using archive etag"`
```

Comment:
The way I implemented this it also applies to metadata json and tilejson responses. Should I remove those? Or make this `--resource-etag`? or just leave it as-is?

Reply:
Also in my tests on a 32-core machine enabling this reduces throughput from 40k tiles per second to 36k tiles per second.

Reply:
maybe `--unique-etags` instead, to make it clearer the effect?